tokio-console/src/conn.rs

```diff
@@ -5,7 +5,11 @@ use console_api::instrument::{
 use console_api::tasks::TaskDetails;
 use futures::stream::StreamExt;
 use std::{error::Error, pin::Pin, time::Duration};
-use tonic::{transport::Channel, transport::Uri, Streaming};
+use tokio::net::UnixStream;
```

Comment:
this import will need to be `cfg`-flagged:
```suggestion
#[cfg(unix)]
use tokio::net::UnixStream;
```